.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '3'
+
+services:
+  app:
+    build:
+      context: ..
+      dockerfile: .devcontainer/Dockerfile
+    ports:
+      - 8000:8000
+    volumes:
+      - ..:/workspace
+      - ~/.ssh:/root/.ssh-localhost:ro
+      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
Why are you exposing the docker socket to the container?

Reply:
This allowed me to access my local docker install from inside the container.

Reply:
Righto, but why?

Reply:
I wanted to be able to also run the commands we have from inside docker, to have the full development experience. From inside the container, I can execute `./api/manage.py makemigrations` which I like. However, I'd also be nice to be able to run `./scripts/makemigrations.sh`.